### DISCUSSION TITLE
How to manage different themes for a same doc ?

### DISCUSSION THREAD

**Discussion by SamuelB-ACA** [outline/outline#2643] [Self Hosting]
Hi,
I have different collections but some documents might belong to more than one. With tree system, I can put my document in only one collection. What about keywords ?
To be more clear (it's just an example)

Collection languages
----Collection Javascript
----Collection PHP

Where should I put a document about "Interaction between javascript and php" ?

If We had keywords, I could :
- add keywords #javascript and #php to my document
- have just 'Collection languages',
- put my documents in this collection. Then I could search "interaction #php"
(or something like this)

Maybe (probably) I'm not doing things the right way but I can't see an other way.

**Comment by tommoor** [comment]
Hey @SamAdum – we have a somewhat long standing request for labels/tags, but this hasn't been implemented yet. 

I understand that labels are good for discovery but you can do exactly what you're suggesting without them, as the search function would find the right documents and you can filter to eg only search the "languages" collection.
